docker/app/entrypoint.sh

```diff
@@ -12,7 +12,8 @@ fi
 /etc/init.d/rsyslog start
 
 # run lfmergeqm on startup to clear out any failed send/receive sessions from previous container
-if [ which /lfmergeqm-background.sh ]; then
+if (which /lfmergeqm-background.sh > /dev/null)
```

Comment:
Per our conversation, the parens for invoking a sub-shell are not necessary here.  But also doesn't seem to affect it.  Remove if you want, but going to approve this PR anyway.

Reply:
I would've preferred to see this conditional removed altogether, seems silly to me (https://github.com/sillsdev/web-languageforge/pull/1034#discussion_r685168994)